airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_32_8_001__AirbyteConfigDatabaseDenormalization.java

```diff
@@ -85,7 +85,7 @@ private static void createEnums(final DSLContext ctx) {
     LOGGER.info("operator_type enum created");
     ctx.createType("namespace_definition_type").asEnum("source", "destination", "customformat").execute();
     LOGGER.info("namespace_definition_type enum created");
-    ctx.createType("status_type").asEnum("active", "inactive", "deprecated").execute();
+    ctx.createType("status_type").asEnum("active", "inactive", "deprecating", "deprecated").execute();
```

Comment:
@jdpgrailsdev Here's an example of where I'm confused about the Jook <-> Migration relationship.  

None of the other file changes in this PR seem to affect the ENUM options in `io.airbyte.db.instance.configs.jooq.generated.enums.StatusType`.  I would have guessed the ENUM options were driven by `airbyte-api/src/main/openapi/config.yaml`.  Failing that, I would have expected Jooq to introspect the columns of the table and load the enums from the postgres ENUM options.  That would mean the right thing to would be to add a new migration altering the ENUM values within postgres to make the change, but that wasn't right either.  Modifying this old migration seems... wrong?

Reply:
@evantahler JOOq generates objects based on the database schema, not the OpenAPI configuration.  This is why this file change affects the generated JOOq objects.  The JOOq code generation does use Flyway to migrate a test database and then use the final schema to generate the objects.